src/Sarif.Multitool.Library/TransformCommand.cs

```diff
@@ -128,6 +128,7 @@ private bool ValidateOptions(TransformOptions transformOptions)
             }
 
             valid &= transformOptions.ValidateOutputOptions();
+            valid &= transformOptions.ValidateOutputFormatOptions();
```

Comment:
>ValidateOutputFormatOptions [](start = 38, length = 27)

I'm wavering between combining these two into one method and leaving it like this.

Reply:
Agree and my opinion is you should combine the two methods into a single validation mechanism, an extension method named ValidateOptions. The fact that users need to remember to call both methods is a clue that we can simplify.

---
In reply to: [500429893](https://github.com/microsoft/sarif-sdk/pull/2100#discussion_r500429893) [](ancestors = 500429893)